webrtc/rtcpeerconnection/rtcconfiguration-icecandidatepoolsize.html

```diff
@@ -0,0 +1,32 @@
+<!doctype html>
+<html>
+<head>
+<title>This test checks that RTCConfiguration.iceCandidatePoolSize is set</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+<script>
+function testICECandidatePoolSize(size, shouldThrow, expectedSize)
+{
+    test(function() {
+        if (shouldThrow) {
+            assert_throws(new TypeError(), () => new RTCPeerConnection({iceCandidatePoolSize: size}))
+            return
+        }
```

Comment:
As a matter of style, I'd like to see throwing and non-throwing tests as two different test-generating functions rather than have one function with a boolean argument.

The functions can have shorter names too - like "testPoolSize(null, 0)" and "testRejectedPoolSize(-1)".

Not a big deal, just my preference for readability.